test/generator/expected_output/compiled_output.coffee

```diff
@@ -12,13 +12,13 @@ class CompiledOutput
             var _v2 = ( ( 5 * 6 ) - _v1 );
             var _v3 = 5.5;
             var _v4 = ( ! true );
-            var _v5 = (function(lb, ub, skip) {
-                var temp = [];
```

Comment:
It's generated code, not the end of the world - but yeah, it's "temp" for a reason, so name it after the reason.